### ISSUE TITLE
🆕 Software Suggestion | nablahost.com

### ISSUE THREAD

**Issue by JonahAragon** [privacytoolsIO/privacytools.io#1043]
## Basic Information

**Name:** NABLA HOST
**Category:** Cloud Storage, Hosting Provider
**URL:** https://nablahost.com/

## Description

**This is my hosting company.** NABLA HOST is providing web hosting and end-to-end encrypted [cloud storage](https://nablahost.com/services/storage/) starting at $0.99/month. We are a fully managed white-glove service, capable of building any custom services as requested.

**Comment by nitrohorse** [comment]
Thanks for sharing, @JonahAragon; this looks really nice. Is there a Terms of Service and Privacy Policy for NABLA’s services?

**Comment by JonahAragon** [comment]
Yes: https://nablahost.com/legal/

**Comment by nitrohorse** [comment]
Nice to see this scores an A+ on [SSL Labs](https://www.ssllabs.com/ssltest/analyze.html?d=nablahost.com). I think nablahost would be valuable to recommend on PTIO.

**Comment by blacklight447-ptio** [comment]
While it does seem nice, wouldn't recommending the services of one of our team members arrouse suspicion/distrust?

**Comment by Mikaela** [comment]
Maybe with a note that it's by a team member?

**Comment by ggg27** [comment]
@JonahAragon This looks really interesting.

Out of curiosity, what OSes do you support?
Also, what do you mean by E2EE for server hosting?

_______________

Might need to switch some of my stuff there, what hardware are you using?

**Comment by JonahAragon** [comment]
@ggg27 yeah, no VPS or dedicated servers yet. Not really our target demographic though, we're more interested in hosting things people otherwise wouldn't know how to do themselves. If you can DIY there's a lot of great server providers out there already 😄

**Comment by ggg27** [comment]
@JonahAragon Still a cool idea though. Even people like me who have some idea of how to manage an Apace Instance probably wouldn't know what to do in case something went wrong or a security flaw was exploited. Managed hosting is basically always better, but is usually expensive.

But, since you are probably just implementing the same things multiple times I would assume you could save time. How it all should be, easy to use.

How do you manage the web apps/sites? Are you using a proprietary service like cPanel or is it [Vesta](https://github.com/serghey-rodin/vesta) or custom software? What software do you use to host it?
